client/js/lti_admin_app.jsx

```diff
@@ -12,7 +12,11 @@ import { getInitialSettings } from './reducers/settings';
 // Polyfill es6 promises for IE
 es6Promise.polyfill();
 
-class Root extends React.Component {
+class Root extends React.PureComponent {
```

Comment:
Changed to React.PureComponent complies with https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prefer-stateless-function.md